test.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.linear_model import LogisticRegression
```

Comment:
did you mean to commit this? It would be cool if you add a tests, but it should go into ``linear_models/tests/test_linearsvc.py`` or something like that and you need to capture the output.

Reply:
check ensemble/tests/test_gradient_boosting.py and the test test_verbose_output there.

Reply:
Ack! No!

Reply:
Sorry that was my hacking just to see how it worked. Was not intended to be committed.

Will remove and add proper tests.

Reply:
I figured ;)